decidim-core/app/cells/decidim/collapsible_list/show.erb

```diff
@@ -7,7 +7,7 @@
         <%= card_for element, cell_options %>
       <% end %>
     <% end %>
-    <span class="card__text--paragraph">
+    <span class="card__text--paragraph collapsible-list__see-more"> <!-- card__text--paragraph collapsible-list__see-more -->
```

Comment:
Maybe this `<!-- ... -->` section should be removeed before merging? 😄

Reply:
Yes, fixed!! Sorry for that @mrcasals !

Reply:
❤️